Build-Dependencies.ps1

```diff
@@ -114,34 +116,25 @@ function Run-Stages {
 }
 
 function Package-Dependencies {
-    if ( $script:PackageName -like 'qt*' ) {
-        $ArchiveFileName = "windows-deps-${PackageName}-${CurrentDate}-${Target}-${Configuration}.zip"
-    } else {
-        $ArchiveFileName = "windows-${PackageName}-${CurrentDate}-${Target}.zip"
-    }
-
-    Push-Location -Stack BuildTemp
-    Set-Location $ConfigData.OutputPath
+    Push-Location -Stack BuildTemp -Path $ConfigData.OutputPath
 
     Log-Information "Cleanup unnecessary files"
-    $Items = @(
-        "./bin/bison.exe"
-        "./bin/libiconv2.dll"
-        "./bin/libintl3.dll"
-        "./bin/m4.exe"
-        "./bin/pcre2*"
-        "./bin/regex2.dll"
-        "./cmake/pcre2*"
-        "./include/pcre2*"
-        "./lib/pcre2*"
-        "./lib/pkgconfig/libpcre2*"
-        "./man1/pcre2*"
-        "./man3/pcre2*"
-        "./share/bison"
-        "./share/doc/pcre2"
-    )
-
-    Remove-Item -ErrorAction 'SilentlyContinue' -Path $Items -Force -Recurse
+
+    switch ( $PackageName ) {
+        dependencies {
+            Get-ChildItem ./bin/*.lib | Move-Item -Destination ./lib
+            Get-ChildItem ./bin/* -Exclude 'libcurl.dll','lua51.*,','Lib','swig.exe' | Remove-Item
+            Get-ChildItem ./cmake/pcre2*,./lib/pcre2* | Remove-Item
+            Remove-Item -Recurse ./lib/pkgconfig
```

Comment:
Are we making an intentional decision to remove all pkgconfig files?

Reply:
Correct, pkg-config is not used outside of Linux/FreeBSD in our case.